frontend/test/resolution/testInteractive.cpp

```diff
@@ -236,6 +242,10 @@ int main(int argc, char** argv) {
       trace = true;
     } else if (0 == strcmp(argv[i], "--scope")) {
       scopeResolveOnly = true;
+    } else if (0 == strcmp(argv[i], "--once")) {
```

Comment:
Noting (late) that these new options aren't documented, but I think it's ok to live with